make/mt-config/mt-presets.mk

```diff
@@ -22,8 +22,8 @@ CONFIG_CC3200                   =memory_fs-control_topic-tls_bsp-secure_file_tra
 CONFIG_CC3200_TLS_SOCKET        =memory_fs-control_topic-tls_socket-secure_file_transfer
 
 # CONFIG for CC3220
-CONFIG_CC3220SF                 =bsp_cc3220sf-memory_fs-tls_bsp
-CONFIG_CC3220SF_TLS_SOCKET      =bsp_cc3220sf-memory_fs-tls_socket
+CONFIG_CC3220SF                 =bsp_cc3220sf-memory_fs-control_topic-tls_bsp_socket-secure_file_transfer
```

Comment:
👍 great update even in mt-presets.mk grats!

tls_bsp_socket --> tls_bsp
tls_bsp_socket works too since in mt-config.mk the string "tls_bsp" is searched for